src/Stratis.Features.FederatedPeg/TargetChain/CrossChainTransferStore.cs

```diff
@@ -231,7 +228,7 @@ private ICrossChainTransfer[] ValidateCrossChainTransfers(ICrossChainTransfer[]
                 if (partialTransfer.Status != CrossChainTransferStatus.Partial && partialTransfer.Status != CrossChainTransferStatus.FullySigned)
                     continue;
 
-                List<(Transaction, TransactionData, IWithdrawal)> walletData = this.federationWalletManager.FindWithdrawalTransactions(partialTransfer.DepositTransactionId);
+                List<(Transaction, IWithdrawal)> walletData = this.federationWalletManager.FindWithdrawalTransactions(partialTransfer.DepositTransactionId);
```

Comment:
@quantumagi ah any chance we can me the tuple variables? than we don't have this weird `Item1` `Item2` scenario.

Reply:
Done.